### PR TITLE
rebasing

### PR DESCRIPTION
**Pull request by winterpk**

### CODE REVIEW

File: app/assets/javascripts/live_entry.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+( function( $ ) {
+
+	/**
+	 * UI object for the live event view
+	 * 
+	 */
+	var liveEntry = {
+
+		init: function() {
+			console.log('doc ready!');
```
Comment (hybridinteractive) [inline app/assets/javascripts/live_entry.js:10]: WTF IS THIS????!!!! lol jk

File: app/assets/javascripts/test.js [hunk 2]
```diff
@@ -1 +0,0 @@
-console.log('winter');
```
Comment (hybridinteractive) [inline app/assets/javascripts/test.js:1]: FS

Comment (winterpk) [inline app/assets/javascripts/test.js:1]: You do see that that is a deleted line right?

File: app/assets/javascripts/live_entry.js (hunk 1 above)
Comment (winterpk) [inline app/assets/javascripts/live_entry.js:10]: I will murder you

**Pull request merged**